src/main/java/uk/gov/gchq/palisade/resource/request/GetResourcesByTypeRequest.java

```diff
@@ -44,8 +45,8 @@ public GetResourcesByTypeRequest type(final String type) {
         return this;
```

Comment:
Use setters

Reply:
lines 42-45 should use setters and not check nulls

Reply:
OK